### PR TITLE
Gamma correct wrapper

### PR DESCRIPTION
**Pull request by jwheeler5**
**Describe your changes**
Adds a plantcv function made by Haley (gamma_correct) that performs gamma correction on the input image. Wraps the skimage gamma correction function

**Type of update**

* New feature or feature enhancement
* Update to documentation

**Associated issues**

**Additional context**

### CODE REVIEW

File: plantcv/plantcv/transform/gamma_correct.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Gamma Correction Function Prototype
```
Comment (HaleySchuhl) [inline plantcv/plantcv/transform/gamma_correct.py:1]: no longer a prototype! :)

**Review by HaleySchuhl — CHANGES_REQUESTED** [review]
Looks great! I added a minor comment about the header on the function, but the one thing missing is adding this function to the updating.md doc page. I updated the branch so it would be ready to merge once that last documentation page is amended.

**Review by HaleySchuhl — APPROVED** [review]
Awesome!

**Review by nfahlgren — APPROVED** [review]
Thanks @jwheeler5!

**Pull request merged**